app/src/main/java/com/example/kit/database/ItemViewHolder.java

```diff
@@ -15,6 +24,46 @@ public ItemViewHolder(ItemListRowBinding binding) {
 
     // TODO: Expand
     public void displayItem(Item item){
+        // Fill in item row values
+
+        DateFormat df = DateFormat.getDateTimeInstance();
         binding.itemNameRow.setText(item.getName());
+        binding.itemDateRow.setText(df.format(item.getAcquisitionDate().toDate()));
+        binding.itemValueRow.setText(item.getValue());
+
+        ArrayList<String> tags = item.getTags();
+
+        Chip[] chips = {
+                binding.tagChip1,
+                binding.tagChip2,
+                binding.tagChip3,
+                binding.tagChip4,
+                binding.tagChip5,
+                binding.tagChip6,
+                binding.tagChip7,
+                binding.tagChip8,
+                binding.tagChip9,
+                binding.tagChip10,
+                binding.tagChip11,
+                binding.tagChip12,
+                binding.tagChip13,
+                binding.tagChip14,
+                binding.tagChip15,
+                binding.tagChip16,
+                binding.tagChip17,
+                binding.tagChip18,
+                binding.tagChip19,
+                binding.tagChip20
+        };
+
+        // Populate chips with tags
+        for(int i = 0; i < tags.size(); i++) {
```

Comment:
would need to loop through all chips, and set any that are index greater than the number of tags for each item to be visibility=GONE just in case the the recyclerview is reusing a viewholder that previously had a greater number of tags than the new item